test/categorical_algebra/DataMigrations.jl

```diff
@@ -318,21 +318,38 @@ X = @acset UndirectedBipartiteGraph begin
   tgt = [1,1,2,3]
 end
 
-Y = ΣF(X)
+Yd = ΣF(X; components=true)
+Y = Graph(codom(Yd))
+α = diagram_map(Yd)
 @test nparts(Y, :V) == 7
 @test nparts(Y, :E) == 4
 @test length(Y[:src] ∩ Y[:tgt]) == 0
+@test isempty(collect(α[:V₁]) ∩ collect(α[:V₂]))
 
 @test SigmaMigrationFunctor(idF, Graph, Graph)(Y) == Y
 
-###Test sigma migrations with attributes
+# Sigma migrations with attributes
+#---------------------------------
+
+# Identity migration on weighted graphs.
+idF = FinFunctor(
+  Dict(VG => VG, EG => EG, :Weight=>:Weight),
+  Dict(srcG => srcG, tgtG => tgtG, :weight=>:weight),
+  SchWeightedGraph, SchWeightedGraph
+)
+Y = @acset WeightedGraph{Symbol} begin
+  V=2; E=2; Weight=1; src=1; tgt=[1,2]; weight=[AttrVar(1), :X]
+end
+ΣF = SigmaMigrationFunctor(idF, WeightedGraph{Symbol}, WeightedGraph{Symbol})
+@test_skip ΣF(Y) == Y
```

Comment:
@KevinArlin, I rebased this PR off of main, which includes #829. I expected that this test would now work, but it fails with an error. Can you please investigate?

Reply:
Yeah, I'll look into it.

Reply:
OK, that should be fixed. It looks like I thought of this attrvar issue briefly while writing this but then forgot to test on it.